src/main/java/com/notably/logic/parser/NewCommandParser.java

```diff
@@ -24,6 +27,7 @@
             + "To see list of Command format, type: help";
 
     private Model notablyModel;
+    private final Logger logger = LogsCenter.getLogger(getClass());
```

Comment:
Oops sorry missed this before. Just a tiny thing. Let's make this `logger` a `private static final` field since we don't need to create one for each instance of `NewCommandParser`!